build-tools/upload-junit-to-cloud.py

```diff
@@ -1,30 +1,43 @@
 from google.cloud import bigquery
 import xml.etree.ElementTree as ET
 import argparse
+import io
+from typing import List, Optional
+import json
 
 # Uploading JUnit test results to BigQuery
-def upload(final: bool):
+def upload(final: bool, url: Optional[str], files: List[io.TextIOWrapper]):
```

Comment:
* info: added support for passing multiple junit xml test files, so we can support both junit and androidtests results. 
* info: added option to pass url, so in case of failure with looker studio, we can trace the test run.